build/buildPackage.sh

```diff
@@ -127,6 +126,15 @@ manifest="
 
 "
 
+if [ "$#" -lt "1" ]; then
```

Comment:
I did start with -gt but it just didn't work. In testing -lt worked but I think my testing was flawed and I assumed it was some bash shell weirdness. I'll flip the sense so the default is to build against renderman unless disabled and figure out the shell scripting required to get this working. 

It's the packaging of IE modules done correctly if renderman doesn't exist? Then the cortex scons build doesn't build the lib & modules and the package looks for IECore* so it handles this correctly?